cloud/pod_util.go

```diff
@@ -171,17 +173,25 @@ func ExportECSContainerResources(info pod.ContainerResourceInfo) cocoa.ECSContai
 	return *res
 }
 
-// ExportPodContainerDef exports the ECS pod container definition into the equivalent cocoa.ECSContainerDefintion.
-func ExportPodContainerDef(opts pod.TaskContainerCreationOptions) (*cocoa.ECSContainerDefinition, error) {
+// ExportPodContainerDef exports the ECS pod container definition into the
+// equivalent cocoa.ECSContainerDefintion.
+func ExportPodContainerDef(settings *evergreen.Settings, p *pod.Pod) (*cocoa.ECSContainerDefinition, error) {
+	script, err := agentScript(settings, p)
+	if err != nil {
+		return nil, errors.Wrap(err, "building agent script")
+	}
 	return cocoa.NewECSContainerDefinition().
-		SetImage(opts.Image).
-		SetMemoryMB(opts.MemoryMB).
-		SetCPU(opts.CPU).
-		SetWorkingDir(opts.WorkingDir).
-		SetEnvironmentVariables(exportEnvVars(opts.EnvVars, opts.EnvSecrets)), nil
+		SetName("evg-agent").
```

Comment:
Should "evg-agent" be a const somewhere?

Reply:
Sure, I made it a constant.